code/modules/mob/living/silicon/robot/death.dm

```diff
@@ -31,6 +31,7 @@
 	canmove = 0
 	icon = null
 	invisibility = 101
```

Comment:
Not caused by this PR, but the `icon` being set to null here makes the ghost of the borg after dusting completely invisible.
Removing this line fixes the issue, so I'm honestly not sure why it was here in the first place.
```suggestion
	canmove = 0
	invisibility = 101
```